capabilities/working-in-small-batches.md

```diff
@@ -1,52 +1,69 @@
+Making many small changes may feel slower than a single large change. But extensive research -- including the DORA reports -- shows that teams that continuously refine and adapt based on real-world usage, rather than upfront assumptions and predictions, achieve faster lead times, higher deployment frequency, lower failure rates, and faster recovery times. Big changes can cause integration problems and delay error discovery, creating unpredictable hiccups. Even if the progress seems slower, making small changes helps teams learn faster, adjust easier, and maintain a steady pace.
 
-A small batch is smallest possible unit of value that can be tested and iterated upon. This means breaking down features into smaller, deployable increments that provide feedback and validate assumptions early. By continuously refining and adapting based on real-world usage rather than upfront assumptions and predictions, teams can reduce wasted effort and mitigate risk of failure.
-
-For example, consider a team building an internal expense reporting tool. Traditionally, they might spend months creating a complete system with receipt scanning, auto-categorization, and advanced approval workflows. A small-batch approach, however, begins with something simpler, like a basic form for entering expenses manually. After releasing this minimal version, the team might quickly learn that managing receipts is the main frustration, prompting them to prioritize adding a straightforward upload feature next. Over time, they may discover features like auto-categorization or complex approvals aren't as critical as originally thought. By delivering smaller increments and improving based on real user feedback, the team provides value sooner and reduces wasted effort.
+For example, consider a team building an internal expense reporting tool. Traditionally, they might spend months creating a complete system with receipt scanning, auto-categorization, and advanced approval workflows. A small-batch approach, however, begins with something simpler, like a basic form for entering expenses manually. After releasing this minimal version, the team might quickly learn that managing receipts is the main user frustration, prompting them to prioritize adding a straightforward upload feature next. Over time, they may discover features like auto-categorization or complex approvals aren't as critical as originally thought. 
 
 ## Nuances
 
 This section outlines common pitfalls, challenges, or limitations teams commonly encounter when applying this capability. The goal here is not to discourage you. Rather, the goal is to arm you with the appropriate context so that you can make an informed decision about when and how to implement the capability with your teams.
 
-### Managing Organizational Resistance
+### Thoughtful, Collaborative Upfront Design
```

Comment:
This subtitle doesn't jump out as a pitfall, challenge, or limitation. Perhaps we rename it to something like, `Tendency to Pass Work Through A Group of Specialists`?

Reply:
What about `Trickle Down Design` or `Incorrect Assumptions About Implementation`?